catroweb.yaml

```diff
@@ -658,7 +660,11 @@ components:
       schema:
         $ref: '#/components/schemas/Uuid'
       required: true
-
+    ProgramId:
+      name: program_id
+      in: query
```

Comment:
would be nice to maybe require it when similar project category is chosen?
not sure if that is possible in swagger

Reply:
at least provide some info text that it should be set else the response will be empty